interface/src/components/grid.js

```diff
@@ -67,6 +79,11 @@ class Grid extends Component{
                 ypos += height; 
             }
         
+            this.setState({
+                label,
+                list
+            })
+
             let grid = d3.select(`#grid${type}`)
             .append("svg")
             .attr("width",26 * mcol)
```

Comment:
Perhaps `.attr("width",(width + 1) * mcol)

Reply:
resolving b/c we will be replacing this w/ leaflet to visualize geojson